### PR TITLE
Imperian wings!

### PR DESCRIPTION
**Pull request by Chimaeran**
Added support for Imperian Orphanim and Seraphim wings. 

 - Added mmp.flying using the 'flying above' that's appended to roomnames. Used for the Imperian wings stuff.
 - Added support for the indoors, outdoors, considered outdoors tags in gmcp.Room.Info.details, so the mmapper went inside and mmapper went outside events trigger for Imperian.
 - The mapper won't consider itself "off path" if it just went from ground to flying when speedwalking now. This uses the mmp.flying stuff and shouldn't interfere with anything.
 
Due to lack of a crowdmap, the list of areas that work with Imperian wings was made by checking which areas have paths to the ruins of Caanae. New areas will need to be manually added.

### CODE REVIEW

File: mudlet-mapper.xml [hunk 1]
```diff
@@ -4835,7 +4835,7 @@ mmp.speedWalkPath = mmp.speedWalkPath or {}
 mmp.speedWalkDir = mmp.speedWalkDir or {}
 
 
-local newversion = &quot;developer&quot;
+local newversion = &quot;18.3.1&quot;
```
Comment (keneanung) [inline mudlet-mapper.xml]: Could you undo this change? The release script relies on the version being `developer`.

File: mudlet-mapper.xml [hunk 2]
```diff
@@ -4882,6 +4882,10 @@ function mmp.startup()
   private_settings[&quot;universe&quot;]      = createOption(false, mmp.setUniverse, {&quot;boolean&quot; }, &quot;Use Universe Tarot?&quot;)
   private_settings[&quot;gare&quot;]      		= createOption(false, mmp.setGare, {&quot;boolean&quot; }, &quot;Use Gare?&quot;)
 
+--Imperian wings
+	private_settings[&quot;shekinah&quot;]     	= createOption(false, mmp.setShekinah, { &quot;boolean&quot; }, &quot;Use Seraphim Wings?&quot;)
+	private_settings[&quot;suriel&quot;]     		= createOption(false, mmp.setSuriel, { &quot;boolean&quot; }, &quot;Use Orphanim Wings?&quot;)
```
Comment (keneanung) [inline mudlet-mapper.xml]: Could you run the code formatter on this script to get rid of the stranhe indenting here?

File: mudlet-mapper.xml [hunk 3]
```diff
@@ -5395,13 +5401,35 @@ function speedwalking( event, num )
   if num ~= mmp.currentroom then mmp.previousroom = mmp.currentroom end
   mmp.currentroom = num
   mmp.currentroomname = getRoomName(num)
+	
+	-- Try to track if we're flying or not, for Imperian wings
+	-- This is to avoid being &quot;off path&quot; if we FLY due to wings.
+	local madeflight=false
+	if gmcp.Char then
```
Comment (keneanung) [inline mudlet-mapper.xml]: Why are you checking for `gmcp.Char` but don't use it? You probably want to check for `gmcp.Room`.

Also, if the formatter works here, could you run it on this script as well?

File: mudlet-mapper.xml [hunk 4]
```diff
@@ -9162,6 +9466,36 @@ end</script>
                             <string>mmp logged in</string>
                         </eventHandlerList>
                     </Script>
+                    <Script isActive="yes" isFolder="no">
+                        <name>mmapper_imperian_went_outside</name>
+                        <packageName></packageName>
+                        <script>function mmapper_imperian_went_outside()
+  if not mmp.autowalking or not (mmp.settings.shekinah or mmp.settings.suriel ) or mmp.game ~= &quot;imperian&quot; or
```
Comment (keneanung) [inline mudlet-mapper.xml]: Please run the formatter on this script as well.

File: mudlet-mapper.xml (hunk 2 above)
Comment (Chimaeran) [reply mudlet-mapper.xml]: I'm not seeing the odd indenting with this script open in mudlet - All looks neat?

Comment (keneanung) [reply mudlet-mapper.xml]: It's a tabs vs. spaces issue here
